sign_oca/wizards/sign_oca_template_generate_multi.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023-2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SignOcaTemplateGenerateMulti(models.TransientModel):
+    _name = "sign.oca.template.generate.multi"
+    _description = "Generate signature requests"
+
+    def compute_model(self):
+        return self.env.context.get("model", False)
+
+    model = fields.Char(
+        readonly=True,
+        default=compute_model,
+        # default=lambda self: self.env.context.get('model', False)
+    )
+    template_id = fields.Many2one(
+        comodel_name="sign.oca.template",
+        domain="[('model', '=', model)]",
```

Comment:
```suggestion
        domain="['|', ('model', '=', False),('model', '=', model)]",
```
This is intentional, you can select templates that do not have a defined model ("_global_") or those that are linked to the model.

Reply:
Hi @victoralmau,

In our case, we are not using the model functionality as it does not apply to our process. Then it seems a bit too invasive to add a 'Sign From Template' to all models without us being able to use it.

I understand that your idea is to have a generic template that is used to sign multiple models, but maybe we could find an intermediate solution. That is, it would be nice to be able to configure a generic template but only in the case that it is intentional. Also, could you confirm if this is the case or if it's something else so I can finally understand it well?

I honestly don't know what would be the best solution for this. I don't know if it would make more sense for the Model field to be a One2many and here you choose the models you want it to apply to. Or maybe add a checkbox that allows you to mark 'Apply to all models'. What do you think?

Reply:
Sorry for the misunderstanding, the idea is as follows:

- Only show the action (with js) if there is some template defined for that model.
- In the wizard allow to select templates of the model or without model (the normal will be to use templates of the model).

This behavior is the same as in v15 and I think it will avoid confusion. What do you think?

Reply:
@victoralmau As it works in previous versions, if you have a template with no model the action will appear everywhere. Why would you want to allow selecting templates that does not have a model defined on it?

In context with what you've said, I think that the best solution should be to remove this _model = False_ because adds a non desired behaviour. And then we would only show the action to models that are defined in a template.